src/schemas/index.ts

```diff
@@ -0,0 +1,3 @@
+import { TBaseField } from "./base-field";
+
+export type TField = TBaseField;
```

Comment:
Don't see a need to assign another name to it. We'll extend the `BaseField` interface if required

Reply:
forgot to mention that `TBaseField` is a placeholder for a type union of future fields